POS/Operation/05_ManageCashRegister.md

```diff
@@ -117,7 +117,7 @@ The shift summary is an overview of all the transactions made within a shift. A
 
```

Comment:
A shift starts when a pay desk is opened and ends when the pay desk is closed. 

(Relative commas sollten weg)

Reply:
done